tests/test_auth.py

```diff
@@ -81,7 +81,7 @@ def test_confirm_email(self, mock_mail):
         res = res.follow()
 
         assert_equal(res.status_code, 302)
-        assert_in('myprojects', res.location)
+        assert_in('/', res.location)
```

Comment:
Minor: this assertion looks too unspecific; almost all paths will contain `/`. Better to check equality. IIRC, you can do

```python
assert_equal(res.request.path, '/')
```